### ISSUE TITLE
No cost a.i.

### ISSUE THREAD

**Issue by SimpleDioney** [jtljrdn/infinite-craft-clone#3]
Could you use llama2? Since it's open source and you can run it locally at no cost...

**Comment by jtljrdn** [comment]
Good Idea. I definitely have thought about it, but I will look more into it.
